course/src/main/java/in/testpress/course/services/VideoDownloadService.kt

```diff
@@ -51,11 +59,13 @@ class VideoDownloadService : DownloadService(
     }
 
     override fun getForegroundNotification(downloads: MutableList<Download>): Notification {
-        // TODO : Add intent and also download message
+
+        val notifyPendingIntent = getNotifyPendingIntent()
```

Comment:
I cannot understand what this variable refers to by seeing its name. Please change it to meaningful name.
Name should explain its intent